net/portmapper/portmapper.go

```diff
@@ -547,16 +590,11 @@ func (c *Client) Probe(ctx context.Context) (res ProbeResult, err error) {
 	} else {
 		uc.WriteTo(pcpAnnounceRequest(myIP), pcpAddr)
 	}
-	if c.sawUPnPRecently() {
```

Comment:
this "saw UPnP recently" thing was a nice optimization we're losing now, though?

Reply:
hm I guess now that I've updated the probe it's fine to add this back, I ripped it out because the old prober didn't work